conbench/entities/benchmark_result.py

```diff
@@ -236,14 +278,38 @@ class BenchmarkResultCreate(marshmallow.Schema):
         marshmallow.fields.Decimal(allow_none=True),
         required=True,
         metadata={
-            "description": "A list of benchmark results (e.g. durations, throughput). This will be used as the main + only metric for regression and improvement. The values should be ordered in the order the iterations were executed (the first element is the first iteration, the second element is the second iteration, etc.). If an iteration did not complete but others did and you want to send partial data, mark each iteration that didn't complete as `null`."
+            # Note(JP): we must specify: is it allowed to send an empty list?
+            # Is it allowed to send a list with only `null` values? Should
+            # there be the requirement that at least one non-null value must be
+            # present?
+            "description": conbench.util.dedent_rejoin(
+                """
+                A list of benchmark results (e.g. durations, throughput). This
+                will be used as the main + only metric for regression and
+                improvement. The values should be ordered in the order the
+                iterations were executed (the first element is the first
+                iteration, the second element is the second iteration, etc.).
+                If an iteration did not complete but others did and you want to
+                send partial data, mark each iteration that didn't complete as
+                `null`."""
+            )
```

Comment:
This is the same text, re-wrapped.

Reply:
Thanks for flagging things like this, makes skipping over so much easier